frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx

```diff
@@ -341,7 +414,7 @@ export const InstallPlanPreview: React.FC<InstallPlanPreviewProps> = ({
   const approve = () =>
     k8sUpdate(InstallPlanModel, { ...obj, spec: { ...obj.spec, approved: true } })
       .then(() => setNeedsApproval(false))
-      .catch(setError);
+      .catch((error) => errorModal({ error: error.message }));
```

Comment:
Safer to use `Object.prototype.toString`:
```suggestion
      .catch((error) => errorModal({ error: error.toString() }));
```